tests/perf/actor_reminder/actor_reminder_test.go

```diff
@@ -39,7 +39,7 @@ const (
 	appName         = "perf-actor-reminder-service"
 
 	// Target for the QPS
-	targetQPS = 50
+	targetQPS = 56
```

Comment:
Why?

Reply:
Because we can now :) the perf has improved a lot so setting a new baseline.